### PR TITLE
[fuzzyclock] add alternative formats for German-speaking regions

### PR DESCRIPTION
**Pull request by IngmarStein**
# Description
There are regional variants for formatting "6:20" (or "6:40"): both "TEN TO HALF SEVEN" and "TWENTY PAST SIX" are acceptable.
Also, some regions use "THREE QUARTERS ELEVEN" instead of "QUARTER TO ELEVEN".

Reference: https://de.wikipedia.org/wiki/Uhrzeit#Sprechweise_im_Deutschen

# Copilot

### Summary
🌐🛠️🕒

This pull request enhances the `fuzzy_clock` app by refactoring the language logic and adding more configuration options. It uses formatted strings instead of dictionaries for each language, and allows users to choose between different patterns for German and Swiss German. It also removes some unnecessary numbers from some languages.

> _The clock of doom is ticking, no mercy for the weak_
> _`fuzzy_clock` app is changing, new languages it will speak_
> _Formatted strings and patterns, unleash the power of the code_
> _Remove the unused numbers, make the clock explode_

### Walkthrough
*  Simplify and customize fuzzy clock patterns for different languages and dialects using formatted strings ([link](https://github.com/tidbyt/community/pull/1584/files?diff=unified&w=0#diff-8fb3dcfaadcf41d62c73625b4184543811a3a96d7cef336fb39226487e3d0d40L65-R228),[link](https://github.com/tidbyt/community/pull/1584/files?diff=unified&w=0#diff-8fb3dcfaadcf41d62c73625b4184543811a3a96d7cef336fb39226487e3d0d40L225-R289))
* Remove unused numbers from Dutch and German dictionaries to save space ([link](https://github.com/tidbyt/community/pull/1584/files?diff=unified&w=0#diff-8fb3dcfaadcf41d62c73625b4184543811a3a96d7cef336fb39226487e3d0d40L34),[link](https://github.com/tidbyt/community/pull/1584/files?diff=unified&w=0#diff-8fb3dcfaadcf41d62c73625b4184543811a3a96d7cef336fb39226487e3d0d40L49-L50))

### CODE REVIEW

**Comment by IngmarStein** [comment]
Friendly ping @matslina

**Comment by betterengineering** [comment]
Hey @mtimkovich! Does this look ok to you?

File: apps/fuzzyclock/fuzzy_clock.star [hunk 1]
```diff
@@ -62,120 +59,192 @@ numbersPerLang = {
...
+        10: "ZEHN\nÜBER\n{hour}",
+        15: "VIERTEL\nÜBER\n{hour}",
+        20: "ZEHN\nVOR HALB\n{next_hour}",
+        25: "FÜNF\nVOR HALB\n{next_hour}",
+        30: "HALB\n{next_hour}",
+        35: "FÜNF\nNACH HALB\n{next_hour}",
+        40: "ZEHN\nNACH HALB\n{next_hour}",
+        45: "VIERTEL\nVOR\n{next_hour}",
+        50: "ZEHN\nVOR\n{next_hour}",
+        55: "FÜNF\nVOR\n{next_hour}",
```
Comment (mtimkovich) [inline apps/fuzzyclock/fuzzy_clock.star:85]: For readability sake I'd prefer if we did something like `"FÜNF,VOR,{next_hour}"` (csv style), and then change L247 to `return pattern.format(hour = cur_hour, next_hour = next_hour).split(",")`

Comment (IngmarStein) [reply apps/fuzzyclock/fuzzy_clock.star]: Done

**Comment by mtimkovich** [comment]
LGTM! 🚢

**Pull request merged**